test/nodejs/constants.js

```diff
@@ -10,7 +10,7 @@ exports.three_test_coordinates = [[7.41337, 43.72956],
 
 exports.two_test_coordinates = exports.three_test_coordinates.slice(0, 2)
 
-exports.test_tile = {'at': [17059, 11948, 15], 'size': 114000};
+exports.test_tile = {'at': [17059, 11948, 15], 'size': 162422};
```

Comment:
I might be seeing ghosts here, but didn't we have inconsistent tile sizes on different machines once? /cc @danpat